hibike/hibike_communicator.py

```diff
@@ -0,0 +1,218 @@
+"""
+   This library contains a set of functions that allow the user to manually send and recieve  messages to and from a device by calling functions
+"""
+
+import hibike_message as hm
+import hibike_process as hp
+import multiprocessing
+import threading
+import time
+
+class HibikeCommunicator:
+  
+   def __init__(self):
+      """
+          Set up the pipes for communication between the device and the Beegle Bum Black, creates a thread to recieve communications from the device, starts up the process that runs the communication, and sets up a dictionary that caches all device values sent to it.
```

Comment:
BeagleBone Black, in general though, please read through the comments and make sure that everything is spelled correctly.